stable/datadog/templates/daemonset.yaml

```diff
@@ -44,7 +48,44 @@ spec:
       {{- if .Values.daemonset.priorityClassName }}
       priorityClassName: {{ .Values.daemonset.priorityClassName }}
       {{- end }}
+      {{- if .Values.systemProbe.enabled }}
+      initContainers:
+      - name: seccomp-setup
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        command:
+        - cp
```

Comment:
What is the purpose of copying the seccomp profile in the kubelet seccomp folder ? Is this agnostic of their setups (path, rights...)?

Reply:
It's described here: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#seccomp for using `localhost/<path>` annotation.